cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -655,13 +655,13 @@ func (p *BicepProvider) deleteDeployment(
 
 func (p *BicepProvider) mapBicepTypeToInterfaceType(s string) ParameterType {
```

Comment:
Do we want to handle the condition of ARM introducing a new type that isn't supported by `azd` more gracefully, i.e. avoiding a panic, and printing an error message? Something like "unable to prompt for unsupported bicep type: '%s'. Provide this value in the parameters file directly, or ensure your version of azd is up to date"

Reply:
I will do a follow up PR to try to clean this up - I need to think a bit more about how we want to model this unknown state - but I would like to unblock the template tests and the C# sample.